src/components/Toggle/Toggle.scss

```diff
@@ -0,0 +1,61 @@
+@import "../../styles/states.scss";
+@import "../../styles/global-css-settings.scss";
+
+@mixin toggle-animation {
+  transition: 100ms $expand-animation-timing;
+}
+
+.monday-style-toggle{
+  &__wrapper {
+    display: flex;
+    align-items: center;
+  }
+
+  &__toggle {
+    @include reset-button();
+    @include toggle-animation;
+    margin-inline: $spacing-small;
+    position: relative;
+    height: 24px;
+    width: 41px;
```

Comment:
shouldn't it be 4x multipications?

Reply:
or at least 2x

Reply:
should be but that's was the figma units... i can double check with evginay

Reply:
either do min-width or have the container decide ?